accord-core/src/main/java/accord/primitives/Timestamp.java

```diff
@@ -19,12 +19,16 @@
 package accord.primitives;
 
 import accord.local.Node.Id;
-import accord.utils.Invariants;
+
+import static accord.utils.Invariants.checkArgument;
 
 import javax.annotation.Nonnull;
 
 public class Timestamp implements Comparable<Timestamp>
 {
+    private static final int REJECTED_FLAG = 0x8000;
```

Comment:
Should `REJECTED_FLAG` be the same as `MERGE_FLAGS`?

Reply:
I will comment. The idea here is that we probably don't want all flags to be mergeable, though today we only have one flag.